src/seedu/addressbook/AddressBook.java

```diff
@@ -451,19 +440,9 @@ private static String getMessageForSuccessfulAddPerson(String[] addedPerson) {
      */
     private static String executeFindPersons(String commandArgs) {
         final Set<String> keywords = extractKeywordsFromFindPersonArgs(commandArgs);
-        final ArrayList<String[]> personsFound = getPersonsWithNameContainingAnyKeyword(keywords);
+        final ArrayList<String[]> personsFound = getPersonsContainingKeyword(keywords);
         showToUser(personsFound);
-        return getMessageForPersonsDisplayedSummary(personsFound);
-    }
-
-    /**
-     * Constructs a feedback message to summarise an operation that displayed a listing of persons.
-     *
-     * @param personsDisplayed used to generate summary
-     * @return summary message for persons displayed
-     */
-    private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> personsDisplayed) {
-        return String.format(MESSAGE_PERSONS_FOUND_OVERVIEW, personsDisplayed.size());
+        return String.format(MESSAGE_PERSONS_FOUND_OVERVIEW, personsFound.size());
```

Comment:
Seems like you are removing a method and inlining it due to it being a one line method. However, by doing this, SLAP may be reduced.